CONTRIBUTING.md

```diff
@@ -61,6 +61,99 @@ with the exception of ommiting argument types in docstrings in favour of type hi
 and class signatures. If you're using a `PyCharm`, you can configure this under
 `File -> Settings -> Tools -> Python Integrated Tools -> Docstrings`.
 
+#### Conventions
+
+- Names of variables, functions, classes and modules should be written between single back-ticks.
+     - ``` A `numpy` scalar type that ```
+     - ``` `X` ```
+     - ``` `extrapolate_constant_perc` ```
+
+- Simple mathematical equations should be written between single back-ticks to facilitate readability in the console.
+     - ``` A callable that takes an \`N x F\` tensor, for ```
```

Comment:
These `\` are not necessary as they show up in the output. I recommend installing [`grip`](https://github.com/joeyespo/grip) and running it locally to check the rendered markdown file, e.g.:
![image](https://user-images.githubusercontent.com/13080878/149815849-0d22b3f8-7ab1-4e63-93b1-bbb9398d5feb.png)